samples/goroutines_100k.go

```diff
@@ -0,0 +1,42 @@
+package main
```

Comment:
I'm not quite clear what this example is demonstrating other than you can start 100K go routines?
IMO the reporting shouldn't happen via forever loop but channels - you do some busy waiting there with `nGoroutines != n` if I understand correctly which is not ideal. But maybe this is too much effort to do via channels.